lib/domain/entity/component/data_object_component.dart

```diff
@@ -150,6 +150,9 @@ class DataObjectComponent with _$DataObjectComponent {
     }
     codeLines.addNewLine();
     final classNamePrefix = fileReference.getTypeDeclaration(DataFileType.none);
+    final objects =
+        _getMapperObjectVariablesContent(MapperType.mapEntityToRequest);
+    final classModifier = objects.isEmpty ? 'const ' : '';
```

Comment:
@cozvtieg9, added logic to the const constructor to have fewer hints in the analyzer